src/js/providers/html5.js

```diff
@@ -443,8 +446,11 @@ define([
                 return;
             }
 
-            _setLevels(item.sources);
-            this.sendMediaType(item.sources);
+            // check if _setLevels has already been called
```

Comment:
https://jwplayer.atlassian.net/browse/JW7-1460
Due to preloading, this is being called twice